SevenZip/ArchiveExtractCallback.cs

```diff
@@ -268,8 +268,11 @@ public int GetStream(uint index, out ISequentialOutStream outStream, AskMode ask
                     if (_actualIndexes == null || _actualIndexes.Contains(index))
                     {
                         var data = new PropVariant();
-                        _archive.GetProperty(index, ItemPropId.Path, ref data);
-                        string entryName = NativeMethods.SafeCast(data, "");
+                        _archive.GetProperty(index,
+                                             ItemPropId.Path,
+                                             ref data);
+                        string entryName = NativeMethods.SafeCast(data,
+                                                                  "");
```

Comment:
The changes to this file would probably be easier to follow without the random (I assume automatically formatted) whitespace and line break changes.

I'll try to read through it anyway, and test it out a bit to see that I understand what's going on :)